cmake/Modules/FindComputeCpp.cmake

```diff
@@ -97,15 +97,29 @@ else()
 endif()
 
 # Obtain the path to the ComputeCpp runtime library
-find_library(COMPUTECPP_RUNTIME_LIBRARY ComputeCpp PATHS ${COMPUTECPP_PACKAGE_ROOT_DIR}
+find_library(COMPUTECPP_RUNTIME_LIBRARY
+  NAMES ComputeCpp ComputeCpp_vs2015
+  PATHS ${COMPUTECPP_PACKAGE_ROOT_DIR}
   HINTS ${COMPUTECPP_PACKAGE_ROOT_DIR}/lib PATH_SUFFIXES lib
   DOC "ComputeCpp Runtime Library" NO_DEFAULT_PATH)
 
 if (EXISTS ${COMPUTECPP_RUNTIME_LIBRARY})
   mark_as_advanced(COMPUTECPP_RUNTIME_LIBRARY)
-  message(STATUS "libComputeCpp.so - Found")
+  message(STATUS "${COMPUTECPP_RUNTIME_LIBRARY} - Found")
 else()
-  message(FATAL_ERROR "libComputeCpp.so - Not found!")
+  message(FATAL_ERROR "ComputeCpp Runtime Library - Not found!")
+endif()
+
+find_library(COMPUTECPP_RUNTIME_LIBRARY_DEBUG
+  NAMES ComputeCpp ComputeCpp_vs2015_d
+  PATHS ${COMPUTECPP_PACKAGE_ROOT_DIR}
+  HINTS ${COMPUTECPP_PACKAGE_ROOT_DIR}/lib PATH_SUFFIXES lib
+  DOC "ComputeCpp Debug Runtime Library" NO_DEFAULT_PATH)
+if (EXISTS ${COMPUTECPP_RUNTIME_LIBRARY_DEBUG})
+  mark_as_advanced(COMPUTECPP_RUNTIME_LIBRARY_DEBUG)
+  message(STATUS "${COMPUTECPP_RUNTIME_LIBRARY_DEBUG} - Found")
+else()
+message(FATAL_ERROR "ComputeCpp Debug Runtime Library - Not found!")
```

Comment:
should this be indented? If I understand this correctly, this is saying that for non-Windows systems, this will look for the ordinary library and set the debug variable to that as well?

Reply:
It should be. Indeed, it does just that. I wanted to minimize to divergent script flow control. I wanted to make the script generally less verbose. At some later point in time I wanted to make the scripts (optionally at least) a bit more silent. Developers don't usually care about things like stock compute++.exe flags.

Reply:
That's fair. I'm not particularly bothered about the output from the script, if at some stage you wanted to remove some of the noisier messages I'd be down to remove them.